sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/client/TokenExchange.java

```diff
@@ -49,25 +65,46 @@ public static synchronized ServiceCall getDbToken(
             TokenExchangeServiceCallback serviceCallback) {
         AppCenterLog.debug(LOG_TAG, "Getting a resource token from App Center...");
         String url = apiUrl + GET_TOKEN_PATH_FORMAT;
-
-        return
-            httpClient.callAsync(
-                url,
-                METHOD_POST,
-                new HashMap<String, String>() { { put(APP_SECRET_HEADER, appSecret); } },
-                    new HttpClient.CallTemplate() {
-
-                        @Override
-                        public String buildRequestBody() {
-                            return buildAppCenterGetDbTokenBodyPayload(partition);
-                        }
-
-                        @Override
-                        public void onBeforeCalling(URL url, Map<String, String> headers) { }
-                    },
-                    serviceCallback);
+        TokenResult cachedToken = TokenManager.getInstance().getCachedToken(partition);
+        if (cachedToken != null) {
+            serviceCallback.callCosmosDb(cachedToken);
+
+            /* Get the token from local cache directly, not need to make http call, build the object and return. */
+            return new ServiceCall() {
+
+                @Override
+                public void cancel() {
+
```

Comment:
remove blank line for empty methods

Reply:
fixed, after merge from that branch again, the method has been changed...